WBooks/AppDelegate.swift

```diff
@@ -35,6 +35,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window?.rootViewController = LoginViewController()
         window?.makeKeyAndVisible()
         
+        // MARK: NavigationBar global styles config
+        UINavigationBar.appearance().tintColor = .white
+        UINavigationBar.appearance().barTintColor = UIColor(hex: "#00ADEF")
```

Comment:
Add all the colors that you use in a `UIColorExtension` and then you can use them like `UIColor.myBeautyColor`